x/auth/tx/service_test.go

```diff
@@ -122,8 +122,15 @@ func (s IntegrationTestSuite) TestSimulateTx_GRPC() {
 			} else {
 				s.Require().NoError(err)
 				// Check the result and gas used are correct.
-				s.Require().Equal(len(res.GetResult().GetEvents()), 6) // 1 coin recv 1 coin spent, 1 transfer, 3 messages.
-				s.Require().True(res.GetGasInfo().GetGasUsed() > 0)    // Gas used sometimes change, just check it's not empty.
+				//
+				// The 13 events are:
```

Comment:
I think the fact that there's only 6 events on `master` is a bug. For one, we added 3 `tx.{acc_seq,signature,fee}` events recently, so it should be at least 9...

@alexanderbez is this maybe related to a recent bug discussed on Discord #verified-validators?

Reply:
Yeah but are we now getting the correct amount?

Reply:
> But are we now getting the correct amount with your changes here?

It seems so yeah, the 13 events all seem correct

Reply:
Nice! Guess we fixed it...somehow :p